flow/flow_test.go

```diff
@@ -1736,6 +1736,35 @@ func TestGREEthernet(t *testing.T) {
 	validatePCAP(t, "pcaptraces/gre-gre-icmpv4.pcap", layers.LinkTypeEthernet, nil, expected)
 }
 
+func TestIperfLocalhost(t *testing.T) {
```

Comment:
Maybe a name describing the purpose of the test and not how the pcap has been generated

```
TestL2L3EqualSrcDst